modules/administration-guide/examples/snip_che-deploy-open-vsx-using-oc.adoc

```diff
@@ -0,0 +1,141 @@
+* Internet connection.
+* Installed oc and git tools.
+* Installed podman.
+* Log in to the OpenShift cluster where a {prod-short} is deployed as an administrator.
++
+[TIP]
+====
+
+`$ oc login pass:c,a,q[{prod-url}] --username=__<my_user>__`
+
+====
+
+.Procedure
+
+. Create a new OpenShift project for Open VSX.
++
+[subs="+attributes,+quotes"]
+----
+oc new-project openvsx
+----
+. Clone Open VSX sorces.
++
+Get Open VSX sources and navigate to the OpenShift deploy directory:
++
+[bash,subs="verbatim",options="nowrap"]
+----
+git clone https://github.com/svor/openvsx.git &&
```

Comment:
This link will be updated once the instructions have been reviewed and validated.